custom_components/tuya_local/devices/klarta_forste4_purifier.yaml

```diff
@@ -91,7 +91,17 @@ entities:
         class: measurement
   - entity: sensor
     translation_key: air_quality
+    class: enum
     dps:
       - id: 21
         type: string
         name: sensor
+        mapping:
+          - dps_val: great
+            value: excellent
+          - dps_val: Good
```

Comment:
Are the values from the device for Good and Poor capitalised? The Klarta Stor may also need modifying in that case, as they are likely to be similar.

Reply:
Yeah, they are. I'm not sure I'd assume that they would be similar :) 
This seems very inconsistent in general. I don't have the Stor though, so I can't verify.
<img width="355" height="488" alt="image" src="https://github.com/user-attachments/assets/925020a2-32af-4f76-9975-42bd4a8d786e" />